docs/install.rst

```diff
@@ -0,0 +1,127 @@
+
+    pip install astrowidgets
+    #
+    # ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
+    # INSTALL NODEJS FROM https://nodejs.org/en/download/
+    # ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑
+    #
+    # (the nodejs on PyPI is *NOT* the right thing)
+    #
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+
+Install with conda
+------------------
+
+conda installation::
+
+    conda install -c conda-forge astrowidgets nodejs
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+    # Optional, but avoids a rebuild when you open lab
+    jupyter labextension build
+
+.. note::
+
+    If you are using a virtual environment and
+    are experiencing problems with the installation, be sure to check and use
+    the suitable pip version. Also, if you are having problem even with the following
+    dependencies installed, please make sure you have the most up-to-date versions.
+    Upgrade dependencies when and where necessary.
+
+The following dependencies are needed *in the kernel of execution*
+to use ``astrowidgets`` in either Jupyter Lab or Notebook. They should be installed automatically when you install astrowidgets:
+
+* ``python >= 3.6``
+* ``numpy``
+* ``astropy``
+* ``ipywidgets>=7.5``
+* ``ipyevents >= 0.6.3``
+* ``ginga>=2.7.1``
+* ``pillow``
+* ``freetype``
+* ``aggdraw``
+* ``jupyterlab>=1``
+* ``opencv`` (optional, not installed by default)
+* ``nodejs`` (only if you plan to use Jupyter Lab, not installed by default)
+
+After installing dependencies, run::
+
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+
+For those using ``conda``, dependencies from the ``conda-forge`` channel
+should be sufficient unless stated otherwise.
+
+Using OpenCV
+------------
+
+If you wish to use `OpenCV <https://docs.opencv.org/master/index.html>`_ to handle the
+drawing in ginga you have two options:
+
+Install OpenCV with pip
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are using pip it looks like the best option is to use the
+`opencv-python` package, which provides pre-built binaries of most of OpenCV::
+
+    pip install opencv-python
+
+However, the `opencv-python project
+<https://github.com/skvark/opencv-python>`_ is quite clear about being
+"unofficial" so you should probably read about the project before using.
+
+Install OpenCV with conda
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This should work on conda::
+
+    conda install -c conda-forge opencv
+
+If, after installing opencv, you get a warning like this::
+
+    astrowidgets/core.py:72: UserWarning: install opencv or set use_opencv=False
+    warnings.warn('install opencv or set use_opencv=False')
+
+then you should try installing a newer version of freetype::
+
+    conda install 'freetype\>=2.10'
+
+For more details, see `this discussion of opencv and astrowidgets
+<https://github.com/astropy/astrowidgets/issues/90>`_.
+
+Widget with Ginga Toolkit
+-------------------------
+
+.. note::
+
+    For vectorized drawing in ``aggdraw``, you can clone
```

Comment:
Is this still necessary, @ejeshcke?

Also, one section with nothing but one note seems incomplete somehow. Any plans to add more text in this section, @mwcraig ?

Reply:
@ejeschke -- is this workaround still necessary?

Reply:
Let me take a look...

Reply:
Yes, and it currently works only for circles.

Reply:
@mwcraig, are you providing a vectorized `Circle` class?  I don't think we added one to Ginga yet.  Didn't we do this as an experimental branch?  Not sure we should mention it until it is available.

Reply:
I do have a branch of `aggdraw` that supports vectorized drawing of objects that Ginga supports.  My hope was that we could refactor the agg renderer in Ginga so that it would be unecessary to create vector objects; i.e. you could just add as many different objects to the Ginga canvas and it would draw them using the vectorized renderer.

Reply:
> @mwcraig, are you providing a vectorized Circle class? I don't think we added one to Ginga yet. Didn't we do this as an experimental branch? Not sure we should mention it until it is available.

Sounds like this should be removed from the documentation for now. I've found performance to be pretty good as long as markers are added in pixel coordinates. I switched the astrowidgets code to convert world coordinates to pixel before marking here https://github.com/astropy/astrowidgets/commit/7c9fb3694029144c5c5be5079ee6ee9e53547d0d

Reply:
Yeah, you will save a round of WCS conversions on redraws if you do that.